code/modules/surgery/organs/subtypes/grey.dm

```diff
@@ -1,3 +1,17 @@
 /obj/item/organ/internal/liver/grey
 	alcohol_intensity = 1.6
 	species = "Grey"
+
+/obj/item/organ/internal/brain/grey
+	icon_state = "brain-x"
+	mmi_icon_state = "mmi_alien"
+	species = "Grey"
+
+/obj/item/organ/internal/brain/grey/insert(var/mob/living/carbon/M, var/special = 0)
+	..()
+	M.add_language("Psionic Communication")
+
+/obj/item/organ/internal/brain/grey/remove(var/mob/living/carbon/M, var/special = 0)
+	..()
+	if("Psionic Communication" in M.languages)
```

Comment:
i feel like there is something horribly wrong with this but i dunno the proper way of checking languages